server/chat-commands/moderation.ts

```diff
@@ -959,7 +953,7 @@ export const commands: ChatCommands = {
 
 	deroomvoiceall(target, room, user) {
 		if (!this.can('editroom', null, room)) return false;
-		if (!room.auth) return this.errorReply("Room does not have roomauth.");
+		if (![...room.auth.entries()].length) return this.errorReply("Room does not have roomauth.");
```

Comment:
```suggestion
		if (!room.auth.size) return this.errorReply("Room does not have roomauth.");
```

lol, `[...map.entries()].length` is certainly one way to write `map.size`, but I do want to point out that `map.size` does exist:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/size